packages/model/index.js

```diff
@@ -10,6 +10,8 @@ module.exports = Object.assign({}, addonBaseConfig, {
   shouldRollupPrivate: true,
   externalDependenciesForPrivateModule() {
     return [
+      'ember-cached-decorator-polyfill',
```

Comment:
the polyfill rewrites the import from `@glimmer/tracking` to be from itself so we declare it here now.